spec/models/fund_spec.rb

```diff
@@ -89,6 +89,18 @@
     end
   end
 
+  describe ".not_ispf" do
+    it "returns a Fund for every entry in the 'fund_types' codelist except for ISPF" do
+      codelist = Codelist.new(type: "fund_types", source: "beis")
+
+      funds = described_class.not_ispf
+      expect(funds.size).to eq(codelist.list.size - 1)
+
+      ids = funds.map(&:id)
+      expect(ids).to eq(codelist.values_for("code") - [4])
```

Comment:
I know we'll be removing this in future so it's not a huge deal, but if this were going to be sticking around for a while, I think I'd rather this were an explicit list of the codelist values we're expecting. The values `- [4]` is pretty much the implementation code itself - we could store them in a `non_ispf_codelists` variable to make it clear what they mean.